src/index.js

```diff
@@ -81,6 +86,7 @@ async function run() {
     const createGithubRelease = core.getBooleanInput('create-github-release');
     const slackChannel = core.getInput('slack-channel');
     const githubChangelogFileDestination = core.getInput('github-changelog-file-destination');
+    const fetchAuthorSlackIds = core.getInput('fetch-author-slack-ids');
```

Comment:
If the `fetch-author-slack-ids` but not `slack-token` we should throw.

Reply:
Added explicit checks for both fetching the users and sending the release notes :+1: